changelogs/unreleased/luarocks-disable-force-config.md

```diff
@@ -1,3 +1,3 @@
 ## feature/luarocks
 
- * set FORCE_CONFIG=false for luarocks config to allow loading project-side .rocks/config-5.1.lua
+ * Set `FORCE_CONFIG=false` for luarocks config to allow loading project-side .rocks/config-5.1.lua.
```

Comment:
While we're at it, it worth to drop the leading whitespace before an asterisk (at least for unification purposes).

Reply:
Sure, thanks for noticing!